addons/mail/views/mail_template_views.xml

```diff
@@ -5,93 +5,100 @@
+                                    <group>
+                                        <separator string="Technical"/>
+                                        <field name="lang" placeholder="{{ object.partner_id.lang }}" options="{'dynamic_placeholder': true}"/>
+                                        <field name="has_mail_server" invisible="1"/>
+                                        <field name="mail_server_id" invisible="not has_mail_server" placeholder="By order of server priority"/>
+                                        <field name="auto_delete"/>
+                                        <field name="scheduled_date" string="Scheduled Send Date" placeholder="Send Instantly" options="{'dynamic_placeholder': true}"/>
+                                    </group>
                                 </group>
                             </page>
-                            <page string="Settings" name="email_settings">
+                            <page string="Options" name="email_settings">
                                 <group col="2">
                                     <group>
-                                        <field name="lang" placeholder="{{ object.partner_id.lang }}"/>
-                                        <field name="mail_server_id"/>
-                                        <field name="auto_delete"/>
-                                        <field name="report_template_ids" domain="[('model','=',model)]"
-                                            widget="many2many_tags"
-                                            options="{'no_create': True}"/>
+                                        <field name="has_dynamic_reports" invisible="1"/>
+                                        <field name="attachment_ids" widget="many2many_binary"/>
+                                        <field name="report_template_ids" widget="many2many_tags"
+                                            domain="[('model','=',model)]"
+                                            invisible="not has_dynamic_reports"
+                                            options="{'no_create': True}"
+                                            placeholder="None"/>
                                     </group>
                                     <group>
                                         <field name="user_id" widget="many2one_avatar_user"
                                             readonly="not is_template_editor"
                                             invisible="not is_template_editor"
-                                            placeholder="Shared with all users."
-                                            help="If set, will restrict the template to this specific user.
-                                                  If not set, shared with all users."/>
-                                        <field name="description"/>
+                                            placeholder="Shared with all users."/>
```

Comment:
Why do some placeholders end with a `.` and others don't? Is there a convention to follow? Just mentioning since it is not consistent.